starboard/shared/starboard/media/codec_util.cc

```diff
@@ -113,7 +113,7 @@ SbMediaAudioCodec GetAudioCodecFromString(const char* codec,
   }
 #endif  // SB_API_VERSION >= 14
 #if SB_API_VERSION >= 15
-  if (strncmp(codec, "iamf.", 5) == 0) {
+  if (strncmp(codec, "iamf", 4) == 0 || strncmp(codec, "iamf.", 5) == 0) {
```

Comment:
Shall `strncmp(codec, "iamf", 4) == 0`  be `strcmp(codec, "iamf") == 0`?

Reply:
done